pages/form/Question5.js

```diff
@@ -1,29 +1,39 @@
 import { useState } from "react";
-import { Heading, Text, Select } from "@chakra-ui/react";
-import Layout from "../../components/Layout/Layout";
-import useForm from "../../components/FormProvider";
-import { departments } from "../../utils/constants";
-import LinkButton from "../../components/LinkButton/LinkButton";
-import capitalize from "../../utils/capitalize";
-import Q5Progress from "../../public/images/progress-bar/q5-progress-bar.svg";
-import { sendFormResponse } from "../../utils/dbApi";
+import { Heading, Text, Select, Stack, Grid } from '@chakra-ui/react';
+import Layout from '../../components/Layout/Layout';
+import useForm from '../../components/FormProvider';
+import { departments } from '../../utils/constants';
+import LinkButton from '../../components/LinkButton/LinkButton';
+import {
+  BackButton,
+  ContinueButton,
+  SubmitButton,
+} from '../../components/LinkButton/LinkButton';
+import capitalize from '../../utils/capitalize';
+import Q5Progress from '../../public/images/progress-bar/q5-progress-bar.svg';
+import { sendFormResponse } from '../../utils/dbApi';
 
 export default function Question5() {
   const { answers, setAnswers } = useForm();
   const [department, setDepartment] = useState(answers.department);
+  const [incentive, setIncentive] = useState(answers.incentive);
 
+  const saveAnswers = () => setAnswers((prev) => ({ ...prev, incentive }));
```

Comment:
`saveAnswers` should be saving the response in `department` variable, not `incentive`; you could inadvertently be overwriting users' response in Question 4 !
```suggestion
  const saveAnswers = () => setAnswers((prev) => ({ ...prev, department }));
```